gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -153,6 +151,26 @@ def _bigquery_to_vcf_shards(
          | vcfio.WriteVcfDataLines())
 
 
+def _form_customized_query(known_args):
```

Comment:
nit: why not just  `_get_bigquery_query`? The 'customized' part is strange as we can have many customizations in the future (e.g. call names, info/format IDs, etc) and those can be in separate 'customized' methods :)